src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +103,52 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search.
```

Comment:
Which of these fields are required? Why allow both search by dataset and read group set?